examples/named_entity_recognition/ner_wikigold_transformer.ipynb

```diff
@@ -28,8 +28,8 @@
     "\n",
```

Comment:
<p>It's better to have this description should occur before the definition of DATA_URL. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>